### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by annikavennemann**
Added a navigation bar using react router.
- Changing color of active nav-icon is still missing

### CODE REVIEW

File: frontend/src/components/Home/Home.js [hunk 1]
```diff
@@ -1,38 +1,256 @@
...
+        font-size: 32px;
+        font-weight: 300;
+        color: #707070
+    }  
 `
+
+const WelcomeBox = styled.div`
+    margin: 1em 0 0 0;
+    padding: 1.5em 2em;
+    box-shadow: 4px 4px 18px #00000029;
```
Comment (bwnstck) [inline frontend/src/components/Home/Home.js:158]: your Hex-Value is not valid :) maybe you just ment `#000` or `#000000` which is the same
max-hex-numbers = 6

File: frontend/src/components/Home/Home.js [hunk 2]
```diff
@@ -1,38 +1,256 @@
...
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+
+    p {
+        text-align: center;
+        font-size: 20px;
```
Comment (bwnstck) [inline frontend/src/components/Home/Home.js:165]: Better use `rem` for fontsizes
```suggestion
        font-size: 1.5rem;
```

File: frontend/src/components/Home/Home.js [hunk 3]
```diff
@@ -1,38 +1,256 @@
...
+const Vision = styled.ul`
+    list-style-image: url("../../images/bulletpoint.svg");
+
+    li {
+        font-size: 20px;
+    }
+`
+
+const ValueWrapper = styled.div`
+    margin: 0 0 0 -0.5em;
```
Comment (bwnstck) [inline frontend/src/components/Home/Home.js:192]: same like above :) makes it more explicit
```suggestion
    margin-right: -0.5em;
```

File: frontend/src/components/Navigation/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    width: 95%;
+    height: 2.5em;
+    display: flex;
+    margin-left: 2.5%;
+    padding: 0.5em 2em;
+    justify-content: space-between;
+    background-color: #FFFFFF;
+    box-shadow: 4px 4px 18px #00000029;
+    border-radius: 10px;
+    z-index: 100;
```
Comment (bwnstck) [inline frontend/src/components/Navigation/Navigation.js:38]: should be sufficient
```suggestion
    z-index: 10;
```

**Review by bwnstck — COMMENTED** [review]
nice one 👩‍🎤

File: frontend/src/components/Navigation/Navigation.js (hunk 4 above)
Comment (annikavennemann) [reply frontend/src/components/Navigation/Navigation.js:38]: Ok, I just wanted to make it VERY clear 😄 but I will change it.

File: frontend/src/components/Home/Home.js (hunk 1 above)
Comment (annikavennemann) [reply frontend/src/components/Home/Home.js:158]: Ohh true, I just copied the css-code of my adobe-xd mockup. thanks for noticing!

**Pull request merged**